src/components/select/select.spec.js

```diff
@@ -153,6 +153,22 @@ describe('<md-select>', function() {
     expect($rootScope.myForm.select.$touched).toBe(true);
   }));
 
+  it('should remain untouched during opening', inject(function($compile, $rootScope) {
+    var form = $compile('<form name="myForm">' +
+                        '<md-select name="select" ng-model="val">' +
+                        '<md-option>1</md-option>' +
+                        '</md-select>' +
+                        '</form>')($rootScope);
+    var unwatch = $rootScope.$watch('myForm.select.$touched',
```

Comment:
Because we want in this test to check, whether mdSelect remains untouched during opening, to prevent ngMessages flicker. The easiest way to do that, is to watch the value of $touched property and assert, that the value is false. Without the assertion in $watch, this test would be the same as the 'should set touched only after closing' test, which was passing previously without the fix.